benchmarks/libjpeg-turbo_libjpeg_turbo_fuzzer/build.sh

```diff
@@ -16,12 +16,10 @@
 set -e
 set -u
 
-cat fuzz/branches.txt | while read branch; do
-    pushd libjpeg-turbo.$branch
-    if [ "$branch" = "main" ]; then
-        sh fuzz/build.sh
-    else
-        sh fuzz/build.sh _$branch
-    fi
-    popd
-done
+cd libjpeg-turbo
+
+# Substitute original fuzz target
+rm fuzz/decompress.cc
+cp ../decompress.cc fuzz/decompress.cc
+
+sh fuzz/build.sh
```

Comment:
Just encountered the same libjpeg-turbo benchmark build failure during my local FuzzBench experiments.

We should also add `FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION` definition in order to overcome the following build failure:

```
#22 16.78 In file included from /src/libjpeg-turbo/fuzz/cjpeg.cc:35:
#22 16.78 /src/libjpeg-turbo/fuzz/../cjpeg.c:700:12: error: no member named 'max_pixels' in 'cjpeg_source_struct'
#22 16.78   src_mgr->max_pixels = 1048576;
#22 16.78   ~~~~~~~  ^
```

The error appears because `max_pixels` field is enabled via the mentioned [define](https://github.com/libjpeg-turbo/libjpeg-turbo/blob/3b19db4e6e7493a748369974819b4c5fa84c7614/cdjpeg.h#L39).

It may be fixed with the following `sed`:

```
sed -i 's/-DNDEBUG/-DNDEBUG -DFUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION/' fuzz/build.sh
```

Reply:
Flag `FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION` should be set by default in fuzzers/utils.py when it initializes other standard compilation flags.

Reply:
Yeah, you are right!